test_scripts/Smoke/Resumption/ATF_Resumption_disconnect_more_than_30s_before_IGN_OFF.lua

```diff
@@ -0,0 +1,114 @@
+--  Requirement summary:
+--  [Data Resumption]: Data resumption on IGNITION OFF
+--
+--  Description:
+--  1. Media App is registered and active on HMI
+--  2. Perform iginition off
+--     Perform ignition on
+--
+--  Expected behavior:
+--  1. SDL store app_info.dat file, App's info is absent in app_info.dat
+--     App is successfully registered and receive default HMI level, persistent data is resumed
+
+-- [[ Required Shared Libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonStepsResumption = require('test_scripts/Smoke/Resumption/commonStepsResumption')
+local mobile_session = require('mobile_session')
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/dummy_connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+-- [[Local variables]]
+local default_app_params = config.application1.registerAppInterfaceParams
+local default_app = nil -- will be initialized after application registration
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+function Test:Start_SDL_With_One_Activated_App()
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self):Do(function()
+    self:initHMI():Do(function()
+      commonFunctions:userPrint(35, "HMI initialized")
+      self:initHMI_onReady():Do(function ()
+        commonFunctions:userPrint(35, "HMI is ready")
+        self:connectMobile():Do(function ()
+          commonFunctions:userPrint(35, "Mobile Connected")
+          self:startSession():Do(function ()
+            commonFunctions:userPrint(35, "App is registered")
+            default_app = self.applications[default_app_params.appName]
+            commonSteps:ActivateAppInSpecificLevel(self, default_app)
+            EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL",audioStreamingState = "AUDIBLE", systemContext = "MAIN"})
+            commonFunctions:userPrint(35, "App is activated")
+          end)
+        end)
+      end)
+    end)
+  end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Transport disconnect >30s before IGNITION_OFF. Media app not resume")
+
+function Test:Close_Session()
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppUnregistered", {unexpectedDisconnect = true, 
+                        appID = self.applications[default_app_params]})
+  self.mobileSession:Stop()
+end
+
+function Test:Sleep_31_sec()
```

Comment:
@LitvinenkoIra Please remove unused parameter `self` that implicitly pass to function